server/blob/blob-objectstorage/src/test/java/org/apache/james/blob/objectstorage/ObjectStorageBlobsDAOTest.java

```diff
@@ -149,11 +150,30 @@ void encryptionWithCustomPayloadCodeCannotBeReadFromUnencryptedDAO() throws Exce
 
     @Test
     void deleteBucketShouldDeleteSwiftContainer() {
-        objectStorageBlobsDAO.deleteBucket(defaultBucketName).block();
-        assertThat(blobStore.containerExists(defaultBucketName.asString()))
+        BucketName bucketName = BucketName.of("azerty");
+        objectStorageBlobsDAO.save(bucketName, "data").block();
+
+        objectStorageBlobsDAO.deleteBucket(bucketName).block();
+
+        assertThat(blobStore.containerExists(bucketName.asString()))
             .isFalse();
     }
 
+    @Test
+    void deleteAllBucketsShouldDeleteAllContainersInBlobStore() {
+        Location defaultLocation = null;
```

Comment:
How is null handled ? Can't we use an optional ?

Reply:
Not my code, I'm late. Sorry.

Reply:
@remk Location is where you expect the container to be created(eg US, UK...), if null, then it is the default location. and null is suppose to be handled in side jclouds library